crates/re_entity_db/src/time_histogram_per_timeline.rs

```diff
@@ -47,20 +52,25 @@ impl TimeHistogramPerTimeline {
     }
 
     #[inline]
-    pub fn num_timeless_messages(&self) -> u64 {
-        self.num_timeless_messages
+    pub fn num_static_messages(&self) -> u64 {
+        self.num_static_messages
+    }
+
+    /// Total number of (non-static) message over all timelines.
+    pub fn num_timeline_messages(&self) -> u64 {
```

Comment:
We use the "static vs. temporal" terminology everywhere else
```suggestion
    /// Total number of temporal message over all timelines.
    pub fn num_temporal_messages(&self) -> u64 {